configure/ai-data-masking/fields.mdx

```diff
@@ -3,7 +3,15 @@ title: "Data Masking Fields"
 description: "Get to know all the fields that you can use to mask data in your organization."
 ---
 
-## Global
+## Microsoft Presidio
```

Comment:
I don't like such direct delegation of responsability from ux perspective. Driving user to unkown territory likely will create big drops in conversion. I would rather have the information shown here (also good for SEO) and then link to source

Reply:
something like: a table with all the fields and checks of supported by presidio vs google dlp would be interesting to see and solve the problem, like:

fields | presidio | google dlp |
-|-|-
brazilian cpf | ✅ | ❌ |